aries_vcx/src/protocols/connection/invitee/state_machine.rs

```diff
@@ -23,7 +23,7 @@ use messages::protocols::connection::request::Request;
 use messages::protocols::connection::response::SignedResponse;
 use messages::protocols::discovery::disclose::{Disclose, ProtocolDescriptor};
 
-#[derive(Clone)]
+#[derive(Clone, Serialize, Deserialize)]
 pub struct SmConnectionInvitee {
     source_id: String,
     thread_id: String,
```

Comment:
as per my other comment, it seems these fields are not being included in serialized form of the Connection